src/backend/aspen/api/schemas/pango_lineages.py

```diff
@@ -0,0 +1,5 @@
+from aspen.api.schemas.base import BaseResponse
+
+
+class PangoLineagesResponse(BaseResponse):
+    all_pango_lineages: list[str]
```

Comment:
can we rename this to `lineages` or something? I'm thinking this endpoint might support filtering at some point, and then the `all_pango_lineages` key name starts to look strange.

Reply:
Yeah, I see your point. I wonder if we should go even farther and change the routing to be more like `/lineages/pango/` because we likely want to support other lineages (like WHO) down the road? Or maybe just `/lineages/` and we would eventually differentiate on more params in the request?

Reply:
I like `/lineages/pango/` actually! I don't expect that it would make sense to support returning lineages from different databases all mixed together, I think it's more likely that they'd have their own endpoints

Reply:
I'm not sure. I expect the most common use-case will be: "Please give me all Pangos, all WHOs, and a mapping that shows how to get WHO<--> Pango". One route that returns that object all at once seems nice, although minorly overloaded.

Reply:
yeah, that sounds like a later-us problem right now?